src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -443,6 +446,21 @@ private void PreInitialize()
                 }
             }
 
+            if (string.IsNullOrEmpty(_environment.GetFunctionsWorkerRuntime()))
```

Comment:
What are everyone's thoughts on this firing in Core Tools? I can make sure the docs explain how to fix this by adding it to `local.settings.json`, but do we think this will cause an outcry? 

I think I'd rather take the hit and get this in front of everyone everywhere we can... but open to toughts.

Reply:
It's just a warning right? So I think it's fine to fire in CT - this way folks can get ahead of it before they even deploy

Reply:
Yes, just a warning.

Reply:
There is also no harm at all for them to add this app setting so I wouldn't expect too much outcry